crates/fuel-core/src/service.rs

```diff
@@ -424,6 +442,10 @@ impl RunnableService for Task {
                 }
             }
         }
+
+        params.block_production_trigger.send_trigger();
+        tracing::info!("Block Production has been triggered.");
```

Comment:
If block production is disabled, this log will be invalid. Maybe we can move it to the PoA service?

Reply:
i'm moving it to the port impl in the poa crate, that's the only place where we know that it has been triggered, otherwise we will have logs for each time `run` is called. chek d16295bbc9